{"uuid": "f0fc8cac-9759-40ae-857e-503d3ecaff78", "children": ["afa91d2b-aed7-4f9a-93e9-4394df524dcb"], "befores": [{"name": "setup_profiles", "status": "failed", "statusDetails": {"message": "Failed: AP latest and active are different\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 78, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_ref,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 565, in setup_profiles\n    pytest.fail(\"AP latest and active are different\")\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/outcomes.py\", line 153, in fail\n    raise Failed(msg=msg, pytrace=pytrace)\n"}, "attachments": [{"name": "Active Config: ", "source": "d081aba5-fa5c-4fd3-9cad-782cb64067f8-attachment.attach"}, {"name": "Latest Config: ", "source": "166530fc-b67b-4b02-a3b1-f2a56142ed11-attachment.attach"}, {"name": "FAILURE: AP Logs: ", "source": "3ef172aa-ef51-4aa7-9885-93d8fdfc1e5e-attachment.attach"}], "start": 1638467439131, "stop": 1638467761276}], "start": 1638467439131, "stop": 1638467762283}